sidecar-injector/README.md

```diff
@@ -126,61 +242,272 @@ For this section, you'll work from a test namespace (test-namespace). Later you
+    ~$ a_test_pod=$(kubectl \
      -n test-namespace \
      get po \
-     -l=app=sleep \
+     -l=app=test-app \
      -o=jsonpath="{.items[0].metadata.name}")
     
     ~$ kubectl \
       -n test-namespace \
-      exec ${a_sleep_pod} \
-      -c sleep \
+      exec ${a_test_pod} \
+      -c app \
       -i \
       -- \
-      bash << 'EOL'
-    export http_proxy=localhost:8080
+      curl --silent localhost:8080 | grep authorization | sed -e s/^authorization=Basic\ // | base64 --decode; echo
+   
+    ```
+    ```
+    "test-secret#username:test-secret#password"
+    ```
     
-    response=$(curl --request GET --url http://scooterlabs.com/echo.json)
-    pretty_resp=$(echo "${response}" | jq -r .headers.Authorization)
-    echo '"'"$(echo "${pretty_resp##* }" | base64 --decode)"'"' | jq .
+
+## Conjur Authenticator/Secretless Sidecar Injection Example
+
+For this section, you'll work from a test namespace (sidecar-example-app). Later you will label this namespace with `cyberark-sidecar-injector=enabled` so as to allow the cyberark-sidecar-injector to operate on pods created in this namespace.
+
+1. Setup a Conjur appliance running with the Kubernetes authenticator installed and enabled - e.g. run `./start` in  [kubernetes-conjur-deploy](https://github.com/cyberark/kubernetes-conjur-deploy/)
+
+1. Load Conjur policy to create a host for the service account `$TEST_APP_SERVICE_ACCOUNT` - e.g. `test-app-secretless` and `test-app-secretless` are made available by walking through [kubernetes-conjur-demo](https://github.com/conjurdemos/kubernetes-conjur-demo) until `./3_init_conjur_cert_authority.sh`
+
+1. Set up environment variables modify to suite your needs e.g. use the same values from [kubernetes-conjur-demo](https://github.com/conjurdemos/kubernetes-conjur-demo)
+    ```bash
+    # required values
+    export TEST_APP_SERVICE_ACCOUNT=test-app-secretless
+    export containerMode=sidecar
+    export CONJUR_VERSION=4
+    export CONJUR_NAMESPACE_NAME=conjur-ktanekha
+    export CONJUR_ACCOUNT=my-account
+    export AUTHENTICATOR_ID=sidecar-test
+    export TEST_APP_NAMESPACE_NAME=sidecar-example-app
+
+
+    # derived values
+    CONJUR_APPLIANCE_URL="https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api"
+    CONJUR_AUTHN_URL="https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api/authn-k8s/${AUTHENTICATOR_ID}"
+    if [ ${CONJUR_VERSION} = '4' ]; then
+      CONJUR_AUTHN_LOGIN=${TEST_APP_NAMESPACE_NAME}/service_account/${TEST_APP_SERVICE_ACCOUNT}
+    else
+      CONJUR_AUTHN_LOGIN=host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${TEST_APP_NAMESPACE_NAME}/service_account/${TEST_APP_SERVICE_ACCOUNT}
+    fi
+    CONJUR_SSL_CERTIFICATE=$(follower_pod_name=$(kubectl -n ${CONJUR_NAMESPACE_NAME} get pods -l role=follower --no-headers | awk '{ print $1 }' | head -1); kubectl exec -n ${CONJUR_NAMESPACE_NAME} $follower_pod_name -- cat /opt/conjur/etc/ssl/conjur.pem)
+    ```
+
+1. Create test namespace
```

Comment:
If you run through the scripts of `kubernetes-conjur-demo` as described above, this namespace will already exist.